editor/plugins/theme_editor_plugin.h

```diff
@@ -419,6 +419,9 @@ class ThemeTypeEditor : public MarginContainer {
 class ThemeEditor : public VBoxContainer {
 	GDCLASS(ThemeEditor, VBoxContainer);
 
+	friend class ThemeEditorPlugin;
```

Comment:
This is not a forward declaration

Reply:
Yes, it's not.
It allows the plugin to assign to itself to `plugin` property.

Reply:
Granted, but the type isn't forward declared when used, that's what I meant, wasn't sure if you just forgot or had it serving both purposes here, sorry

Reply:
It compiled for me, so I thought the forwarding isn't needed. But I guess it's SCU thing again.

Reply:
Yeeah, SCU is tricky